modules/_labs/core/react/README.md

```diff
@@ -106,7 +106,7 @@ information in the [theming documentation](./lib/theming/README.md).
 
 ```tsx
 import * as React from 'react';
-import {CanvasProvider} from '@workday/canvas-kit-react';
+import {CanvasProvider} from '@workday/canvas-kit-react-common';
```

Comment:
The `Providers`, `Theming`, and `Bidirectionality` sections in this readme should move to common. Make sure to update the links when you do.